src/main/java/core/basesyntax/service/Parser.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+
+public interface Parser {
+    FruitTransaction parse(String line);

```

Comment:
```suggestion
public interface Parser {
    List<FruitTransaction> parse(List<String> lines);
```
What about doing more work in Parser and less work in the main() method?